### PR TITLE
More Mycus Monsters 2: Electric Boogaloo

### PR DESCRIPTION
**Pull request by zdlpoppa**
#### Summary
Content "Adds more fungalized humanoid/animal zombie variants."

#### Purpose of change
Fungal creature variety is simply underwhelming currently, being able to encounter different types of fungal enemies/friends would make the faction/race a bit more interesting.

#### Describe the solution
Mainly adds more monster blocks, while adding the appropriate "fungalize_into": for normal humanoid/animal monsters so the monsters can appear properly in game.

#### Describe alternatives you've considered
Not making the fungal fellas.

#### Testing
A few test runs on my own world, haven't encountered any errors yet.

#### Additional context
This is the 2nd version of More Mycus Monsters, and my first technical contribution. Any additional critiques/help would be greatly appreciated!

### CODE REVIEW

File: data/json/monsters/fungus_zombie.json [hunk 1]
```diff
@@ -352,5 +352,469 @@
...
+    },
+    "burn_into": "mon_zombie_scorched",
+    "flags": [ "SEES", "HEARS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "PUSH_MON", "FILTHY" ],
+    "armor": { "bash": 6, "cut": 8, "bullet": 6, "electric": 1 }
+  },
+  {
+    "id": "mon_zombie_dog_fungus",
+    "type": "MONSTER",
+    "name": { "str": "fungal dog" },
+    "description": "The deformed, animated, and again reanimated corpse of a what looked like the concept of a canine.  It's lost almost all of it's dog-like features, except for its constant aggressive growling, and tendency to bite.  It shakes its fur like a wet dog, sending tiny spore clouds everywhere.",
```
Comment (TheShadowFerret) [inline data/json/monsters/fungus_zombie.json:589]:
```suggestion
    "description": "The deformed, animated, and again reanimated corpse of a what looked like the concept of a canine.  It's lost almost all of its dog-like features, except for its constant aggressive growling, and tendency to bite.  It shakes its fur like a wet dog, sending tiny spore clouds everywhere.",
```

File: data/json/monsters/fungus_zombie.json [hunk 2]
```diff
@@ -352,5 +352,469 @@
...
+    "death_drops": "default_zombie_death_drops",
+    "burn_into": "mon_zombie_scorched",
+    "flags": [ "SEES", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "PUSH_MON", "FILTHY" ],
+    "armor": { "electric": 1 }
+  },
+  {
+    "id": "mon_zombie_rot_fungal",
+    "type": "MONSTER",
+    "name": { "str": "fungal rotter" },
+    "description": "The rotten skin that flakes from the host's body seems to be covered in tiny blue dots; occasional popping and crackling sounds are heard, like cracking wood.  this corpse can't be discerned as human by the naked eye.",
```
Comment (NetSysFire) [inline data/json/monsters/fungus_zombie.json:403]:
```suggestion
    "description": "The rotten skin that flakes from the host's body seems to be covered in tiny blue dots; occasional popping and crackling sounds are heard, like cracking wood.  This corpse can't be discerned as human by the naked eye.",
```

File: data/json/monsters/fungus_zombie.json [hunk 3]
```diff
@@ -352,5 +352,469 @@
...
+    },
+    "burn_into": "mon_zombie_scorched",
+    "flags": [ "SEES", "HEARS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "PUSH_MON", "FILTHY" ],
+    "armor": { "bash": 6, "cut": 8, "bullet": 6, "electric": 1 }
+  },
+  {
+    "id": "mon_zombie_dog_fungus",
+    "type": "MONSTER",
+    "name": { "str": "fungal dog" },
+    "description": "The deformed, animated, and again reanimated corpse of a what looked like the concept of a canine.  It's lost almost all of its dog-like features, except for its constant aggressive growling, and tendency to bite.  It shakes its fur like a wet dog, sending tiny spore clouds everywhere.",
```
Comment (NetSysFire) [inline data/json/monsters/fungus_zombie.json:589]:
```suggestion
    "description": "The deformed, animated, and again reanimated corpse of a what looked like the concept of a canine.  It has lost almost all of its dog-like features, except for its constant aggressive growling, and tendency to bite.  It shakes its fur like a wet dog, sending tiny spore clouds everywhere.",
```

File: data/json/monsters/fungus_zombie.json [hunk 4]
```diff
@@ -352,5 +352,469 @@
...
+    "color": "green",
+    "scents_tracked": [ "sc_human", "sc_fetid" ],
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 5,
+    "melee_damage": [ { "damage_type": "cut", "amount": 6 } ],
+    "dodge": 2,
+    "families": [ "prof_intro_biology", "prof_wp_zombie" ],
```
Comment (Maleclypse) [inline data/json/monsters/fungus_zombie.json:676]: Just noticed you aren't using the fungal weakpoints from #69583.

**Review by Maleclypse — DISMISSED** [review]

File: data/json/monsters/fungus_zombie.json (hunk 4 above)
Comment (Maleclypse) [reply data/json/monsters/fungus_zombie.json:676]: Otherwise this looks really close to merging.

**Pull request merged**